k8s-artifacts/k8s_configmaps.yaml

```diff
@@ -0,0 +1,259 @@
+# config maps to be mounted for deployments in wso2 Envoy micro gateway
```

Comment:
Lets add license headers.

Reply:
We need to have separate files for each configmap, deployment service. 

Structure is like this..


K8s-artifacts/mg-router/deployment.yaml
                                        /service.yaml
                                        /configmap.yaml

We need a similar folder structure here.

Reply:
Configmaps can be named as config-toml-confimap.yaml, log4j-configmap.yaml

Reply:
Fixed in 736f7ca09587d301737d3a6bff36ff140493ae4b